src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let unitedString = '';
+
+    for (let i = 0; i < this.length; i++) {
+      const validation
+      = this[i] === undefined
+      || this[i] === null
+      || this[i] === '';
+
+      const addValue = validation ? '' : this[i];
+
+      unitedString += i === this.length - 1
+        ? addValue
+        : addValue + String(separator);
```

Comment:
Isn't separator string from arguments?

Reply:
![image](https://user-images.githubusercontent.com/91015967/211509603-e7db0a04-ac33-4175-92c2-2744a6914adc.png)

It should work with null as separator

![image](https://user-images.githubusercontent.com/91015967/211510158-695e0bb8-4ee9-4c3f-94de-980bdabfb96f.png)

Reply:
👍